### PR TITLE
Fix: Issue 663 social links

### PR DESCRIPTION
**Pull request by nfpappas-oddball**
# Description
addresses feedback from the UX design team. We wanted to make sure we sent the right link to social media when shared out. 

## Ticket
Relates to [Issue 663](https://github.com/department-of-veterans-affairs/next-build/issues/663)

## Developer Task

```[tasklist]
- [x] PR submitted against the `main` branch of `next-build`.
- [x] Link to the issue that this PR addresses (if applicable).
- [x] Define all changes in your PR and note any changes that could potentially be breaking changes.
- [x] PR includes steps to test your changes and links to these changes in the Tugboat preview (if applicable).
- [x] Provided before and after screenshots of your changes (if applicable).
- [x] Alerted the #accelerated-publishing Slack channel to request a PR review.
- [x] You understand that once approved, you are responsible for merging your changes into `main`. (Note that changes to `main` will move automatically into production.)
```

## Testing Steps
Locally:

1. start yarn dev
2. from the home page go to the footer
3. click outreach events
4. click an event
5. use the element inspector on the "add to google", "share on facebook", and "share on twitter" links
6. verify that the href contains the link given the current testing environment.

In DEV
I published the code for this change to the development environment to test it. The result was that the links work as expected.
image

## Screenshots

Before:
image

After:
image
# Reviewer

### Reviewing a PR

This section lists items that need to be checked or updated when making changes to this repository. 

## Standard Checks

```[tasklist]
- [ ] Code Quality: Readabilty, Naming Conventions, Consistency, Reusability
- [ ] Test Coverage: 80% coverage
- [ ] Functionality: Change functions as expected with no additional bugs
- [ ] Performance: Code does not introduce performance issues
- [ ] Documentation: Changes are documented in their respective README.md files
- [ ] Security: Packages have been approved in the TRM
```

## Merging an Approved Layout

When merging a layout, you must ensure that the content type has been turned on for `next-build` inside the `CMS`. This CMS flag must be turned on for editors to preview their work using the next build preview server.

Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved.

The status of layouts should be kept up to date inside [templates.md](./templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.

### CODE REVIEW

**Comment by nfpappas-oddball** [comment]
As I created this PR, I noticed that VA.gov prod shares the old functionality currently. We may want to address the change because we are ignoring the current paradigm. We may also want to verify that this has the desired outcome.

**Review by mreed101 — APPROVED** [review]

**Review by mreed101 — CHANGES_REQUESTED** [review]
I noticed that the social links do not open in a new tab

**Comment by nfpappas-oddball** [comment]
> I noticed that the social links do not open in a new tab

I noticed that in tugboat, we fail to get the url. I see that it looks like a bad replacement ${TUGBOAT_DEFAULT_SERVICE_URL}. I may need @timcosgrove  or someone who knows tugboat to buy in. I can push the change to dev and test it there if we want.

File: src/data/queries/newsStoryTeaser.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const formatter: QueryFormatter<NodeNewsStory, NewsStoryTeaser> = (
     headingLevel: 'h2', //@todo fix headingLevel,
     title: entity.title,
     image: queries.formatData('media--image', entity.field_media), //cropType: '2_1_large'
-    link: entity.path.alias,
+    link: `${entity.path.alias}`,
```
Comment (timcosgrove) [inline src/data/queries/newsStoryTeaser.ts:26]: Is this necessary here?

**Review by timcosgrove — APPROVED** [review]

**Review by mreed101 — APPROVED** [review]

**Pull request merged**